src/new-apps/components/AppActivateDialog/AppActivateDialog.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import Decorator from "@saleor/storybook/Decorator";
```

Comment:
you don't have to create story for it, we want to clean up storybook anyway, only pages will land there

Reply:
this applies also to the other stories you created

Reply:
I copied/partially copied some components from /apps catalog to make later removal of /apps with all its components for replacements with /new-apps less painful, also stories. 

I can remove it, but now, there are two things:
1. We should create stories for only pages, not smaller components, it's fine
2. Dialogs are mostly rendered in views and whether they have to be displayed is mostly controlled via query params

So, now, how can we treat the 2. point, if storybook pages don't display them (in current code structure), I believe we also want to include them in storybook?

Reply:
ok, I see, basically, we should be able to also trigger them on pages, but it's ok keep it for now, don't want to complicate your task here, it's out of scope